imod_coupler/drivers/dfm_metamod/dfm_wrapper.py

```diff
@@ -58,6 +59,24 @@ def get_waterlevels_1d(self) -> NDArray[np.float_]:
             all_waterlevels[nr_nodes_2d : nr_nodes_2d + nr_nodes_1d], dtype=np.float_
         )
 
+    def get_internal_node_coordinates(self) -> NDArray[np.double]:
```

Comment:
should this function be private? then you can start the name with 2 underscores
https://www.geeksforgeeks.org/private-methods-in-python/